### PR TITLE
enhance register modal

### PR DESCRIPTION
**Pull request by temptemp3**
- enhance overview register modal
  - direct user to rerun voi swarm install script to go online
  - add advanced directions 
- enable delegate access and add contract picker

## Summary by CodeRabbit

- **New Features**
	- Introduced a new `Banner` component to enhance UI across various pages.
	- Added a `StakingForecast` component for users to forecast staking rewards.
	- Implemented an acknowledgment step in the `Deposit` component to inform users about the implications of their actions.
	- Expanded the `Overview` component with new data displays, including estimated rewards and total staked amounts.
	- Added a new tab for "Staking Forecast" in the dashboard for easy navigation.
	- Introduced a `DeadlineCountdown` component to display countdown timers for deadlines.
  
- **Bug Fixes**
	- Updated routing in the `AppRouter` for improved navigation clarity.
	- Adjusted validation logic for deposit amounts to reflect new minimum balance requirements.

- **Chores**
	- Added a new dependency (`react-syntax-highlighter`) to support syntax highlighting.
	- Enhanced styling in the `Deposit` component for better user experience.
	- Improved styling for the Staking Forecast page to enhance user interaction.

### CODE REVIEW

File: apps/web/src/Pages/Overview/Register/Register.tsx [hunk 1]
```diff
@@ -51,9 +59,36 @@ function Register({
...
+    }
+
+    if (accountData.contractAddress !== parentAddress) {
+      showSnack(
+        "Invalid parent address (must match contract address)",
+        "error"
+      );
+      return;
+    }
+
```
[the remark this answers is not in the document]
Comment (temptemp3) [reply apps/web/src/Pages/Overview/Register/Register.tsx:91]: No

File: apps/web/src/Pages/Overview/Deposit/Deposit.scss [hunk 2]
```diff
@@ -44,3 +44,51 @@
...
+        text-transform: uppercase;
+        background-color: #007bff;
+        color: #fff;
+
+        &:hover {
+          background-color: #0056b3;
+        }
+      }
+    }
+  }
```
[the remark this answers is not in the document]
Comment (temptemp3) [reply apps/web/src/Pages/Overview/Deposit/Deposit.scss:93]: No

File: apps/web/src/Pages/Overview/Withdraw/Withdraw.scss [hunk 3]
```diff
@@ -44,3 +44,51 @@
...
+      margin-bottom: 20px;
+      font-size: 16px;
+      color: #333;
+
+      div {
+        padding: 10px;
+        background-color: #fff;
+        border-radius: 4px;
+        border: 1px solid #ccc;
+      }
```
[the remark this answers is not in the document]
Comment (temptemp3) [reply apps/web/src/Pages/Overview/Withdraw/Withdraw.scss:75]: No

File: apps/web/src/Pages/Overview/Overview.tsx [hunk 4]
```diff
@@ -44,7 +44,10 @@ import humanizeDuration from "humanize-duration";
 import { InfoTooltip } from "../../Components/InfoToolTip/InfoToolTip";
 import { Copy } from "lucide-react";
 import CopyText from "@/Components/Copy";
-import Register from "../Stake/Register/Register";
+import Register from "./Register/Register";
+import Banner from "@/Components/Banner/Banner";
+import axios from "axios";
+import moment from "moment";
```
[the remark this answers is not in the document]
Comment (temptemp3) [reply apps/web/src/Pages/Overview/Overview.tsx:50]: No

**Review by hsoerensen — APPROVED** [review]

**Pull request merged**